art/estimators/classification/scikitlearn.py

```diff
@@ -35,6 +35,7 @@
     ClassGradientsMixin,
     ClassifierMixin,
 )
+from art.estimators.regression.scikitlearn import ScikitlearnDecisionTreeRegressor
```

Comment:
@beat-buesser Are you sure that ScikitlearnDecisionTreeRegressor should be used within the get_trees method of ScikitlearnGradientBoostingClassifier?

Reply:
@abigailgold Yes, I think that should be correct. It looks like the trees of the ensemble in `sklearn.ensemble.GradientBoostingClassifier` are of type `DecisionTreeRegressor` in https://github.com/scikit-learn/scikit-learn/blob/2beed55847ee70d363bdbfe14ee4401438fba057/sklearn/ensemble/_gb.py#L195